### ISSUE TITLE
Using GA for optimization of CNN model

### ISSUE THREAD

**Issue by iamc1oud** [renatoosousa/GeneticAlgorithmForFeatureSelection#2]
I am doing research on image classification. As in this code you have used LogisticRegression. Can i use Keras or tensorflow cnn model to optimize it's accuracy?
Thank you.

**Comment by renatoosousa** [comment]
I have no experience with such libraries, but I think spending a little time is possible. If you can, please share with us.
